capstone-app/src/app/modules/history/wiki-media/wiki-media.component.html

```diff
@@ -15,7 +16,26 @@
       <a href="{{url.key}}" target="_blank">{{url.value}}</a>
     </p>
   </mat-card>
+  <button mat-icon-button [matMenuTriggerFor]="menu">
+    <mat-icon>language</mat-icon>
+    <span>Languages</span>
+  </button>
+  <mat-menu #menu="matMenu" class="languages">
+    <button *ngFor="let lang of languages | keyvalue" mat-menu-item (click)="onChangeLanguage(lang.key)">
```

Comment:
Wouldn't this set the list of buttons to "en, es, fr,... " etc. Can you use a map to have the names be "English, Spanish, French.." etc?